sec.club/src/components/Loader/Loader.scss

```diff
@@ -1,9 +1,13 @@
+@import "../../variables.scss";
+
 .overlay {
 	position: fixed;
 	top: 0;
 	left: 0;
 	right: 0;
 	bottom: 0;
+	background-color: $background-color;
+	z-index: -1;
```

Comment:
I don't think this is a good idea. It makes the class name `overlay` deceptive

Reply:
Put this in `App.scss` instead
```scss
body {
  background-color: $background-color;
}
```

Reply:
This solution doesn't fix the problem I described though. I'm not sure why, but a change that was made makes the background of the loader white instead of the background color of the rest of the site

Reply:
It fixes it for me